src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const onlyMen = people.filter(man => man.sex === 'm');
+
+  const menFiltered = century
+    ? onlyMen.filter(man => Math.ceil(man.died / 100) === century)
+    : onlyMen;
+
+  return +(menFiltered.reduce((sum, obj) => sum + (obj.died - obj.born), 0)
```

Comment:
you do this twice, so, don't forget about DRY - don't repeat yourself
So, make a separate function for this

Reply:
it still not fixed, create a separate function for reduce